packages/blockchain-utils-validation/src/index.ts

```diff
@@ -1,11 +1,11 @@
 import { LinkProof } from '@ceramicnetwork/blockchain-utils-linking'
-import ethereum from './blockchains/ethereum'
-import filecoin from './blockchains/filecoin'
-import polkadot from './blockchains/polkadot'
-import eosio from './blockchains/eosio'
-import cosmos from './blockchains/cosmos'
-import near from './blockchains/near'
-import tezos from './blockchains/tezos'
+import { handler as ethereum } from './blockchains/ethereum.js'
```

Comment:
optional nitpick: any way we could give these a better default name?

Reply:
We call this thing a handler, and the current structure has proven to be easy enough for developers to grok and extend, so please, let it remain it as it is, unless you have some better ideas.